hypertrace-core-graphql-attribute-scope/src/main/java/org/hypertrace/core/graphql/atttributes/scopes/HypertraceCoreAttributeScopeString.java

```diff
@@ -0,0 +1,7 @@
+package org.hypertrace.core.graphql.atttributes.scopes;
+
+public interface HypertraceCoreAttributeScopeString {
+  String SPAN = "EVENT";
+  String SPAN_EXTERNAL = "SPAN";
```

Comment:
SPAN vs SPAN_EXTERNAL?

Reply:
I briefly commented on this here, but maybe it's worth elaborating: https://github.com/hypertrace/hypertrace-core-graphql/blob/856a1f63489cbcd46cf31bc517fb99fdb1259fbf/hypertrace-core-graphql-common-schema/src/main/java/org/hypertrace/core/graphql/common/schema/attributes/AttributeScope.java#L13
The current graphql scheme doesn't quite correspond one to one with the scopes used in the platform. Span is one example of that, with the name "SPAN" in the graphql schema corresponding to "EVENT" in the rest of the platform. There are others at the HT level I believe.

So our options to resolve this are:
1. Rename them in graphql (SPAN -> EVENT)
1. Rename them internally (EVENT -> SPAN)
1. Bridge them in gql (what this is doing)

Ideally, I'd like to do 2, but that would be much more involved. It might be feasible to do that with the shift to scope strings though. In the meantime, I did 3 to prevent any breaking changes in this PR.

Reply:
ok.

Reply:
Thinking a bit more on this, the constant can actually be removed for now. The external representation is already represented by the enum name (which is used in the incoming deser direction), so using this value as a separate constant for the outgoing ser direction is just confusing. The whole enum itself will go away once we 
1. Transition the callers to read the string version. This is enabled by this PR
2. Transition the callers to write the string version. This is not yet implemented, but will necessitate some kind of external constant and string/string converter if we're still bridging from (SPAN -> EVENT). There's only a small number of places where scopes (or entity types, or trace types) are passed in today.